# viperpy-data
Data Repository for (VIPER)[]<sup>1</sup> analysis in Python, used in the Tutorials for the <code style="color : green">pyVIPER package</code>. 

All data in this repository were sampled, postprocessed and generated from [Peng et al., 2019](https://www.nature.com/articles/s41422-019-0195-y#Sec26) <sup>2</sup>.

# Directories:
- `metadata`: collects metadata for UMI matrices and gene expression signatures.
  <code style="color : green">Describe each file</code>

- `networks`: collection of regulatory networks inferred with the [ARACNe3](https://www.mdpi.com/1099-4300/25/3/542)<sup>3</sup>, the latest implementation of the [ARACNe](https://www.nature.com/articles/ng1532)<sup>4</sup> algorithm.
  <code style="color : green">Describe each file</code> 

- `umi-matrices`: collects UMI matrices used in [Tutorial 1]() and [Tutorial 2]() <code style="color : green">add urls Tutorials</code>. These UMI matrices are subset of selected cell populations from  data in [Peng et al., 2019](https://www.nature.com/articles/s41422-019-0195-y#Sec26) <sup>2</sup>
  <code style="color : green">Describe each file</code> 

- `gex_signatures`: gene expression signatures to be used with VIPER. See [Tutorial 1]() and [Tutorial 2]() <code style="color : green">add urls Tutorials</code> for details.
  <code style="color : green">Describe each file</code> 


## Citation
<code style="color : green">Add text</code>

## References
1. Griffin, A. T., Vlahos, L. J., Chiuzan, C. & Califano, A. NaRnEA: An Information Theoretic Framework for Gene Set Analysis. Entropy 25, 542 (2023). https://doi.org/10.3390/e25030542
2. Peng, J., Sun, BF., Chen, CY. et al. Single-cell RNA-seq highlights intra-tumoral heterogeneity and malignant progression in pancreatic ductal adenocarcinoma. Cell Res 29, 725–738 (2019). https://doi.org/10.1038/s41422-019-0195-y## References
3. Basso, K., Margolin, A., Stolovitzky, G. et al. Reverse engineering of regulatory networks in human B cells. Nat Genet 37, 382–390 (2005). https://doi.org/10.1038/ng1532

