# viperpy-data
Data Repository for VIPER analysis<sup>1</sup> in Python. These data are used in the Tutorials of the ${\color{green}PyVIPER-and-url}$ package

All data were sampled, postprocessed and generated from Peng et al., 2019 <sup>2</sup>.

# Directories:
- `metadata`: collects metadata for UMI matrices and gene expression signatures.

  ${\color{green}Describe--each--file}$

- `networks`: collection of regulatory networks inferred with ARACNe3, the latest implementation of the ARACNe<sup>3,4</sup> algorithm.

  ${\color{green}Describe--each--file}$

- `umi-matrices`: collects UMI matrices used in [Tutorial 1]() and [Tutorial 2](). ${\color{green}Add--url--Tutorials}$ These UMI matrices are subset of selected cell populations from  data in Peng et al., 2019<sup>2</sup>

  ${\color{green}Describe--each--file}$

- `gex_signatures`: gene expression signatures to be used with VIPER. See [Tutorial 1]() and [Tutorial 2]() ${\color{green}Add--url--Tutorials}$ for details.

  ${\color{green}Describe--each--file}$


## Citation
${\color{green}Add--url--Tutorials}$

## References
1. Alvarez, M., Shen, Y., Giorgi, F. et al. Functional characterization of somatic mutations in cancer using network-based inference of protein activity. Nat Genet 48, 838–847 (2016). https://doi.org/10.1038/ng.3593
2. Griffin, A. T., Vlahos, L. J., Chiuzan, C. & Califano, A. NaRnEA: An Information Theoretic Framework for Gene Set Analysis. Entropy 25, 542 (2023). https://doi.org/10.3390/e25030542
3. Peng, J., Sun, BF., Chen, CY. et al. Single-cell RNA-seq highlights intra-tumoral heterogeneity and malignant progression in pancreatic ductal adenocarcinoma. Cell Res 29, 725–738 (2019). https://doi.org/10.1038/s41422-019-0195-y## References
4. Basso, K., Margolin, A., Stolovitzky, G. et al. Reverse engineering of regulatory networks in human B cells. Nat Genet 37, 382–390 (2005). https://doi.org/10.1038/ng1532

